README.adoc

```diff
@@ -60,6 +60,13 @@ $ minishift addons --enable <addon-name>
 
 For more information about enabling, disabling, and applying add-ons, see the link:https://docs.openshift.org/latest/minishift/using/addons.html[Add-ons] topic in the Minishift documentation.
 
+[[ci-setup]]
+== CI Setup
+
+`minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+link:https://ci.centos.org/job/minishift-addons-nightly-b2d/[Nightly job] runs daily at midnight.
```

Comment:
Running PR job is little tricky as I am thinking to run `make integration ADDON=<addon-name>`.
It might take time.
But we should nightly as soon as possible.

Reply:
Can we get the branch name in jenkins? Then we could ask the contributors to make the branch name same as the add-on name and we can get it to jenkins. Maybe there are easier ways to do it.. just a thought.

Reply:
@agajdosi @anjannath Implemented :) See https://github.com/minishift/minishift-addons/pull/99#issuecomment-375748232